### DISCUSSION TITLE
New Windows 11 Sound flyout

### DISCUSSION THREAD

**Discussion by sharath-5br2r** [valinet/ExplorerPatcher#1660] [Ideas]
Insider builds of windows contains a new sound flyout with mixer will the windows 10bflyout can be replaced like wifi flyout

**Comment by pyrates999** [comment]
That can already be replaced by setting the registry with this value: https://winaero.com/enable-old-volume-control-in-windows-10/

You can also change it by opening up the explorer patcher properties application and go to system tray.  Look for the sound option.

**Reply by sharath-5br2r** [reply]
i mean new volume control
![Screenshot 2023-07-29 115415](https://github.com/valinet/ExplorerPatcher/assets/56215847/373721a4-c491-4223-8bde-f029c2b78910)

**Comment by pyrates999** [comment]
From what I can tell, the devs only test explorer patcher against releases, not insider builds and preview.
